### PR TITLE
[MAT-1774] Update bundle parser to handle single base64 encoded library contents

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
The MAT has updated its tooling to base64 encode library contents. Removing the extra base64 decode from the bundle parser.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: [MAT-1774](https://jira.cms.gov/browse/MAT-1774)
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Review by danmee10 — APPROVED** [review]

File: lib/measure-loader/mat_measure_files.rb [hunk 1]
```diff
@@ -96,6 +96,12 @@ def verify_library_versions_match(cql, elm, id, version)
         end
       end
 
+      def decode_content(content)
```
Comment (adongare) [inline lib/measure-loader/mat_measure_files.rb:99]: is this method being used somewhere?

Comment (jkotanchik-SB) [reply lib/measure-loader/mat_measure_files.rb:99]: No... It's bandaid code I wrote to handle both single and double encoded content.

**Review by serhii-ilin — APPROVED** [review]

File: lib/measure-loader/mat_measure_files.rb (hunk 1 above)
Comment (jkotanchik-SB) [reply lib/measure-loader/mat_measure_files.rb]: Gone now.

**Review by adongare — APPROVED** [review]

**Review by adongare — APPROVED** [review]
